docs/index.html

```diff
@@ -226,21 +236,15 @@ <h1 id="error"></h1>
       return settings
    }
 
-   var query = window.location.search.substring(1)
-   var settings = _.chain(query.split('&')).map(function(params) {
-      var p = params.split('=')
-      return [p[0], decodeURIComponent(p[1])]
-   }).fromPairs().value()
-   settings = extendWithDefaults(settings)
-
+   // main start
    if (settings && settings.mode && settings.url && settings.token) {
       start(settings)
    } else {
       var settingsForm = document.querySelector('.settings-form')
       var modeSelect = document.getElementById('mode')
       var serverUrlInput = document.getElementById('serverurl')
       var apiTokenInput = document.getElementById('apitoken')
-      var submit = document.getElementById('submit')
+      const form = document.querySelector('form');
```

Comment:
We have kept away from `const` in order to provide all the possible running environments.

Reply:
I think that we can look at this from a couple of different perspectives. This is the one I looked at it when writing the comments:

- All the current radiators are deployed "somewhere" with "some setup" that is not known to maintainers of this repository.
- Most of them are forks so they can opt-in to an update like this. We have also provided a "hosted" service as a homepage for this repository on Github Pages.
- We have had the non-ES6 level of browser support from the beginning
- Transition like this might not be backwards compatible for some
- Gains from transitioning to ES6 would benefit only the developers in this project – development alltogether is very sporadic and rare

I think that the (arguably miniscule) risk of breaking compatibility is more important than having an extra drop of elegance for the codebase.